modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1154,8 +1177,14 @@ Besides the default `--help`, `-f`/`--fields` and `-o`/`--output` options, this
 [%header,cols="30a,40a,30a"]
 |===
 |Command | Description |  Example
-|`--name` | New asset name .2+<.<| `exchange asset modify --name newName --tags test,sample`
-|`--tags` | Comma-separated tags for the asset
+|`--name` | Asset name (required) .2+<.<|
```

Comment:
we should not change this part

Reply:
Exactly, this is asset modify. We shouldn't change this. 

Additionally, this table is formatted incorrectly.

![image](https://user-images.githubusercontent.com/62706868/117006816-4453b680-acbf-11eb-9caa-7c4e594151da.png)

Reply:
@sbehrends Should the asset modify changes be discarded?

Reply:
@LucasLudueno , @constanzaderienzo  - should we discard the changes to asset modify? I fixed the table formatting. Can you re-review? Thanks.

Reply:
Correct, the changes to `asset modify` should be discarded. The only change should be to add the `uploadv2` command.